xc7/tests/install_test/Makefile

```diff
@@ -0,0 +1,39 @@
+TOP:=top
+VERILOG:=counter_basys3.v
+DEVICE:=xc7a50t_test
+BITSTREAM_DEVICE:=artix7
+PCF=basys.pcf
+
+all: ${TOP}.bit
+
+${TOP}.eblif:
+	synth -t ${TOP} -v ${VERILOG}
+
+${TOP}.net: ${TOP}.eblif
+	pack -e ${TOP}.eblif -d ${DEVICE}
+
+${TOP}.place: ${TOP}.net
+	place -e ${TOP}.eblif -d ${DEVICE} -p ${PCF} -n ${TOP}.net
+
+${TOP}.route: ${TOP}.place
+	route -e ${TOP}.eblif -d ${DEVICE}
+
+${TOP}.fasm: ${TOP}.route
+	write_fasm -e ${TOP}.eblif -d ${DEVICE}
+
+${TOP}.bit: ${TOP}.fasm
+	write_bitstream -d ${BITSTREAM_DEVICE} -f ${TOP}.fasm -b ${TOP}.bit
+
+clean:
+	rm -f *.rpt
```

Comment:
Rather than removing lots of files, can you put the files in a build directory, and then just delete that?

Reply:
fixed